MetaPhase-Consulting/State-TalentMAP
# Show alert on potential session expire

Show an alert when all of the following occur at once:
- The user is using Go Browser
- The session has been active for at least 1 hour
- An `undefined` type axios network failure has occurred
- The alert has not already been displayed

One way to test this:
- Change `isOnProxy` function to check for `"localhost"` insteadof `"msappproxy"`
- Decrease timeout time from `hoursToMilliseconds(1)` to a shorter wait time
- Try to perform a network request with Offline network simulated in Chrome
